### DISCUSSION TITLE
Is it possible I upload my OSSF Scorecard results from Gitlab to the https://api.securityscorecards.dev API dashboard?

### DISCUSSION THREAD

**Discussion by rbalinas** [ossf/scorecard#4188] [Q&A]
Hello,

Relatively new to DevOps, but I have been tasked with researching implementing OSSF Scorecard to our enterprise repo. I am able to replicate running OSSF Scorecard to my non enterprise GitHub repo and post results to the api dashboard, which, when enabling GitHub scorecard actions is pretty straightforward, but I wanted know if the same results from gitlab can be published to the api dashboard so we can see the results. 

To summarize: I have a private gitlab repo, I can successfully run ossf scorecard, publish results and store results as an .json artifact. But can I somehow post those results to the scorecard api dashboard?

Example of the OSSF Scorecard readme documentation on how to show the scorecard score in the README.md.
#[![OpenSSF Scorecard](https://api.securityscorecards.dev/projects/gitlab.com/example-repo/badge)](https://securityscorecards.dev/viewer/?uri=gitlab.com/example-repo/})

Any assistance/guidance would be greatly appreciated.

**Comment by spencerschrock** [comment]
There's two parts here.
1. GitLab equivalent of the GitHub Scorecard Action. This doesn't currently exist. The closest related issue would be [this one](https://github.com/ossf/scorecard-webapp/issues/561).
2. Private repos. We don't upload scores to our public API if it comes from a private GitHub repo. If you're only looking to visualize, the "viewer" file is [here](https://github.com/ossf/scorecard-webapp/blob/main/scorecards-site/static/viewer/index.html) and you may be able to modify it for your own purposes to read from a local file.

**Reply by rbalinas** [reply]
Thank you so much for the information. This is very helpful. Do you know of any particular tools that other developers have used as a means of visualizing the "viewer" file? Initial research is pointing towards possibly Power BI and Tableau to display the results from the json artifact.
